### PR TITLE
Adds properties command

### PR DESCRIPTION
**Pull request by rosenhouse**

### CODE REVIEW

**Comment by xoebus** [comment]
Nice! Thank you. I left one small comment inline.

File: commands/properties.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	properties, err := container.Properties()
+	failIf(err)
+
+	if command.AsJSON {
+		toPrint, err := json.MarshalIndent(properties, "", "  ")
+		failIf(err)
+		os.Stdout.Write(toPrint)
+	} else {
+		for k, v := range properties {
+			fmt.Printf("%s:%s\n", k, v)
```
Comment (xoebus) [inline commands/properties.go]: Could these please be tab separated instead?

**Comment by rosenhouse** [comment]
Fixed!

**Comment by xoebus** [comment]
I think the string formatting lost a percent by accident!

**Comment by rosenhouse** [comment]
Oy.  Don't drink & code, kids.  Lets try that again.

**Comment by xoebus** [comment]
Thanks!

**Pull request merged**